pkg/apis/apps/validation/validation.go

```diff
@@ -377,6 +375,25 @@ func ValidateRollingUpdateDaemonSet(rollingUpdate *apps.RollingUpdateDaemonSet,
 	return allErrs
 }
 
+// ValidateRollingUpdateStatefulSet validates a given RollingUpdateStatefulSet.
+func ValidateRollingUpdateStatefulSet(rollingUpdate *apps.RollingUpdateStatefulSetStrategy, fldPath *field.Path) field.ErrorList {
```

Comment:
Do not export this function, unless there is usage that I missed.

Reply:
fixed